include/boost/geometry/algorithms/detail/calculate_sum.hpp

```diff
@@ -30,7 +34,7 @@ class calculate_polygon_sum
     template <typename ReturnType, typename Policy, typename Rings, typename Strategy>
     static inline ReturnType sum_interior_rings(Rings const& rings, Strategy const& strategy)
     {
-        ReturnType sum = ReturnType();
+        ReturnType sum = ReturnType(0);
```

Comment:
Yes, we're not consistent across the library.

We talked about a similar thing some time ago (also with @brunolalande) and AFAIK he also expressed that if you want to represent specific numeric value then you should rather assign it explicitly. See the discussion here: https://github.com/boostorg/geometry/pull/197#discussion-diff-23688564

And I'd argue that currently the number of explicit initializations with 0 is greater than value initializations. Just try the following grep searches:

`[identifier] = 0`
`grep -rE '[_A-Z0-9a-z]+ \= 0' include/`
Found in `algorithms, rtree, formulas, strategies, util`.
Types `CT, calculation_type, robust_calcualtion_type, coordinate_type`, etc.

`[identifier] = [identifier](0)`
`grep -rE '[_A-Z0-9a-z]+ \= [_A-Z0-9a-z]+\(0\)' include/`
Found in `algorithms, formulas, strategies, util`.
Types `Distance, return_type, coordinate_type, CoordinateType, T`

`[identifier] = [identifier]()`
`grep -rE '[_A-Z0-9a-z]+ \= [_A-Z0-9a-z]+\(\)' include/`
Found in `algorithms, strategies, util`
Types `area_result_type, ReturnType, coordinate_type, distance_type`, etc.